helix-term/src/keymap.rs

```diff
@@ -607,15 +607,17 @@ impl Default for Keymaps {
                 "D" => goto_first_diag,
                 "f" => goto_prev_function,
                 "c" => goto_prev_class,
-                "p" => goto_prev_parameter,
+                "a" => goto_prev_parameter,
+                "p" => move_prev_para,
```

Comment:
cc @sudormrfbin I changed `p` to `a` for parameter because parameter are also known as argument (although the meaning is slightly different) but paragraph have no alternatives, the other reason being paragraph is a known motion in both vim and kakoune, this can keep the consistency and make it easier to learn.